addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -128,7 +128,7 @@
                             </Anagrafica>
                         </DatiAnagrafici>
                         <t t-call="l10n_it_edi.account_invoice_it_FatturaPA_sede">
-                            <t t-set="partner" t-value="record.commercial_partner_id"/>
+                            <t t-set="partner" t-value="record.partner_id"/>
```

Comment:
I think you told me, but I still wonder e.g. in cases where you have subdepartments, ...  It would still be good to check the original ticket.  (which I should have included)